.github/workflows/build.yml

```diff
@@ -40,15 +40,15 @@ jobs:
           toolchain: ${{ matrix.rust }}
 
       - name: cargo build (debug; default features)
-        run: cargo build
+        run: cargo build --locked
 
       - name: cargo test (debug; all features)
-        run: cargo test --all-features
+        run: cargo test --locked --all-features
         env:
           RUST_BACKTRACE: 1
 
       - name: cargo build (debug; rustls-provider-example)
```

Comment:
provider-example and examples are part of the workspace and thus use the workspace `Cargo.lock`. connect-tests and fuzz aren't part of the workspace. (I don't remember why this is the case for connect-tests -- I think because we didn't want to run the tests by default, but not including it in default-members might be better for that? -- for fuzz it is because it requires nightly).

Reply:
> provider-example and examples are part of the workspace and thus use the workspace Cargo.lock. connect-tests and fuzz aren't part of the workspace.

Aha :bulb: Ok that makes sense. Thanks for the explanation